src/components/Swiper/Swiper.tsx

```diff
@@ -0,0 +1,116 @@
+                        modules={[FreeMode, Navigation, Thumbs]}
+                        className="mySwiper2"
+                    >
+                        <SwiperSlide>
+                            <img src={ImagePath.Product} />
+                        </SwiperSlide>
+                        <SwiperSlide>
+                            <img src={ImagePath.Product} />
+                        </SwiperSlide>
+                        <SwiperSlide>
+                            <img src={ImagePath.Product} />
+                        </SwiperSlide>
+                        <SwiperSlide>
+                            <img src={ImagePath.Product} />
+                        </SwiperSlide>
+                        <SwiperSlide>
+                            <img src={ImagePath.Product} />
+                        </SwiperSlide>
+                    </Swiper>
+                    <Swiper
+                        // onSwiper={setThumbsSwiper}
+                        loop={true}
+                        spaceBetween={10}
+                        slidesPerView={4}
+                        freeMode={true}
+                        watchSlidesProgress={true}
+                        modules={[FreeMode, Navigation, Thumbs]}
+                        className="mySwiper"
+                    >
+                        <SwiperSlide>
+                            <img src={ImagePath.Product} />
+                        </SwiperSlide>
+                        <SwiperSlide>
+                            <img src={ImagePath.Product} />
+                        </SwiperSlide>
+                        <SwiperSlide>
+                            <img src={ImagePath.Product} />
+                        </SwiperSlide>
+                        <SwiperSlide>
+                            <img src={ImagePath.Product} />
+                        </SwiperSlide>
+                        <SwiperSlide>
+                            <img src={ImagePath.Product} />
+                        </SwiperSlide>
+                    </Swiper>
+                </Box>
+                </Container>
+                    </Grid>
+                    <Grid item xs={6} md={4}>
+                        <Box>
+                        <Span>SKELETON HAND SHIRT</Span>
+                        <P>$24.59</P>
+                        <Subspan>COLOR</Subspan>
+                            <Buttons background={'red'} ></Buttons>
+                            <Buttons background={'yellow'} ></Buttons>
+                            <Buttons background={'green'} ></Buttons>
+                            <Buttons background={'#66'} ></Buttons>
+                            <Buttons background={'#92a039'} ></Buttons>
+                            <Buttons background={'#59a5de'} ></Buttons>
+                            <Buttons background={'#b0ff00'} ></Buttons>
+                            <Buttons background={'#2bf1b6'} ></Buttons>
+                            <Buttons background={'#9126f8'} ></Buttons>
```

Comment:
You can use an array of colors and map through them instead of dublicating the components